lumicks/pylake/kymo.py

```diff
@@ -8,7 +8,7 @@
 from .detail.timeindex import to_timestamp
 
 
-class Kymo(BaseScan, ConfocalImage):
+class Kymo(ConfocalImage, BaseScan):
```

Comment:
I decided to go with making `ConfocalImage` a subclass of `BaseScan` for a couple reasons:
- many of the methods in `ConfocalImage` depend on the attributes initialized in `BaseScan` so it makes sense to make the relationship concrete
- I think it's best to keep `_plot_color()` as a not-implemented method in `BaseScan` because the other plotting functions defined there depend on it explicitly. So subclassing is really the only way to get around the MRO issue in this case